File: windycityswing/android/WindyCitySwing/app/src/main/java/com/windycityswing/windycityswing/Calendar.java
package com.windycityswing.windycityswing;

import android.graphics.Typeface;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

public class Calendar extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_calendar);

        TextView title = findViewById(R.id.title);
        Typeface titleTypeface = Typeface.createFromAsset(getAssets(), "fonts/DancingScript-Regular.otf");
        title.setTypeface(titleTypeface);

        TextView subtitle = findViewById(R.id.subtitle);
        Typeface subtitleTypeface = Typeface.createFromAsset(getAssets(), "fonts/Rampung.ttf");
        subtitle.setTypeface(subtitleTypeface);
    }
}

File: windycityswing/android/WindyCitySwing/app/src/main/java/com/windycityswing/windycityswing/Calendar.kt
package com.windycityswing.windycityswing

import android.graphics.Typeface
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView

class Calendar : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_calendar)

        val title = findViewById(R.id.title) as TextView
        val titleTypeface = Typeface.createFromAsset(assets, "fonts/DancingScript-Regular.otf")
        title.typeface = titleTypeface

        val subtitle = findViewById(R.id.subtitle) as TextView
        val subtitleTypeface = Typeface.createFromAsset(assets, "fonts/Rampung.ttf")
        subtitle.typeface = subtitleTypeface
    }
}
